inst/templates/bs4_book.html

```diff
@@ -34,6 +34,33 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script>
 
   <!-- CSS -->
+  $if(csl-css)$
+  <style type="text/css">
+    /* for pandoc --citeproc since 2.11 */
+    div.csl-bib-body { }
+    div.csl-entry {
+      clear: both;
+    $if(csl-entry-spacing)$
+      margin-bottom: $csl-entry-spacing$;
+    $endif$
+    }
+    .hanging div.csl-entry {
```

Comment:
This should be `.hanging-indent` instead of `.hanging` (at least with latest Pandoc 2.17.1.1).

Reply:
Thanks I'll update this.

Reply:
Latest Pandoc is having the same as here though 🤔 
https://github.com/jgm/pandoc/blob/581c94913fa055fa0d925db893991f01647b37f6/data/templates/styles.html#L187-L190

Hanging indent is defined elsewhere in the template 
https://github.com/jgm/pandoc/blob/581c94913fa055fa0d925db893991f01647b37f6/data/templates/styles.html#L168

We could add this too though.

Reply:
And I forgot this was also discussed previously above https://github.com/rstudio/bookdown/pull/1277#discussion_r737289032

Reply:
Ok, thanks for the hints.

Just for the record, this is the relevant bibliography div generated by bookdown (latest master):

```html
<div id="bibliography" class="section level1 unnumbered">
  <h1>Bibliography<a class="anchor" aria-label="anchor" href="#bibliography"><i class="fas fa-link"></i></a>
  </h1>
  <div id="refs" class="references csl-bib-body hanging-indent">
    <div id="ref-citationkey1" class="csl-entry">
      ...
    </div>
    <div id="ref-citationkey2" class="csl-entry">
      ...
    </div>
    <div id="ref-citationkey3" class="csl-entry">
      ...
    </div>
  </div>
</div>
```

None of the elements has the `.hanging` class as you can see.

Reply:
Yes I know - Bibliography part is generated by Pandoc and this how Pandoc does thing. So maybe there is a case (I don't know which) where this is required. With specific CSL attributes maybe ? IDK. 
 
You could open an issue there if you thing this is incorrect or a duplicate as there is also `div.hanging-indent`. Or ask in pandoc discuss. 

IMO it is better for **rmarkdown** to not change Pandoc on this, especially if putting the same as in template is working.